extensions/ql-vscode/src/view/method-modeling/MethodModelingView.tsx

```diff
@@ -1,14 +1,20 @@
 import * as React from "react";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 import { MethodModeling } from "./MethodModeling";
 import { ModelingStatus } from "../model-editor/ModelingStatusIndicator";
 import { ExternalApiUsage } from "../../model-editor/external-api-usage";
+import { ToMethodModelingMessage } from "../../common/interface-types";
 
 export function MethodModelingView(): JSX.Element {
+  const [method, setMethod] = useState<ExternalApiUsage | undefined>(undefined);
+
   useEffect(() => {
     const listener = (evt: MessageEvent) => {
       if (evt.origin === window.origin) {
-        // Nothing to do yet.
+        const msg: ToMethodModelingMessage = evt.data;
+        if (msg.t === "setMethod") {
+          setMethod(msg.method);
+        }
```

Comment:
```suggestion
        } else {
          assertNever(msg.t);
        }
```
Should we add a `assertNever(msg.t)` here to ensure we continue to implement all message types.